test/integration_test.py

```diff
@@ -928,7 +936,7 @@ def _assertResponseStatus(self,
         )
 
     def _check_compact_manifest(self, _catalog: CatalogName, response: bytes):
```

Comment:
Now that the parameter is actually used, it no longer needs to begin with an underscore